docker-compose-tests.yaml

```diff
@@ -19,6 +42,6 @@ services:
       - /bin/bash
       - -c
       - |
-        ./control.sh init
+        bundle exec rake db:create
```

Comment:
Why do we not continue to use the `./control.sh init` and then make sure that part is up-to-date, so we make sure to generate the database equally in our docker tests and locally?

Reply:
Docker tests have also been changed to use Rake. I believe I would rather that we use "native" Rake commands instead of the added abstraction of the control script

Reply:
Then we should also remove the command from `./control.sh` if we do not use it any more. Either remove all of the old references or reuse the old references to avoid code duplication